inc/Widget/SmailyWidget.php

```diff
@@ -155,46 +155,194 @@ public function form( $instance ) {
+		$email_field_placeholder			= isset( $instance['email_field_placeholder'] ) ? $instance['email_field_placeholder'] : __( 'Email', 'smaily' );
+		$name_field_placeholder				= isset( $instance['name_field_placeholder'] ) ? $instance['name_field_placeholder'] : __( 'Name', 'smaily' );
+		$submit_button_text					= isset( $instance['submit_button_text'] ) ? $instance['submit_button_text'] : __( 'Send', 'smaily' );
+		$submit_button_color				= isset( $instance['submit_button_color'] ) ? $instance['submit_button_color'] : '';
+		$submit_button_text_color			= isset( $instance['submit_button_text_color'] ) ? $instance['submit_button_text_color'] : '' ;
+		$use_site_submit_button_color		= isset( $instance['use_site_submit_button_color'] ) ? $instance['use_site_submit_button_color'] : 'default_background_color';
+		$use_site_submit_button_text_color	= isset( $instance['use_site_submit_button_text_color'] ) ? $instance['use_site_submit_button_text_color'] : 'default_text_color';
+		$current_autoresponder				= isset( $instance['autoresponder'] ) ? json_decode( $instance['autoresponder'], true ) : array( 'id' => null );
 		?>
-		<!--  Title -->
-		<p>
-		<label for="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>">
-			<?php esc_attr_e( 'Title:', 'smaily' ); ?>
-		</label> 
-		<input 
-			class="widefat" 
-			id="<?php echo esc_attr( $this->get_field_id( 'title' ) ); ?>" 
-			name="<?php echo esc_attr( $this->get_field_name( 'title' ) ); ?>" 
-			type="text" value="<?php echo esc_attr( $title ); ?>">
-		</p>
-
-		<!-- Autoresponder -->
-		<?php if ( ! empty( $autoresponder_list ) && ! array_key_exists( 'empty', $autoresponder_list ) ) : ?>
-		<p>
-		<label for="<?php echo esc_attr( $this->get_field_id( 'autoresponder' ) ); ?>">
-			<?php esc_attr_e( 'Autoresponder:', 'smaily' ); ?>
-		</label>
-		<select id="<?php echo esc_attr( $this->get_field_id( 'autoresponder' ) ); ?>" name="<?php echo esc_attr( $this->get_field_name( 'autoresponder' ) ); ?>" class="widefat" style="width:100%;">
-			<?php
-			// Show selected autoresponder.
-			if ( ! empty( $instance['autoresponder'] ) ) {
-				$current_autoresponder = json_decode( $instance['autoresponder'], true );
-				echo '<option value="' . $instance['autoresponder'] . '">' . $current_autoresponder['name'] . '</option>';
-			}
-			// Show all autoresponders from Smaily.
-			if ( ! empty( $autoresponder_list ) && ! array_key_exists( 'empty', $autoresponder_list ) ) {
-				echo '<option value="">' . esc_html__( '-No Autoresponder-', 'smaily' ) . ' </option>';
-				foreach ( $autoresponder_list as $autoresponder ) {
-					echo '<option value="' . htmlentities( json_encode( $autoresponder ) ) . '">' . $autoresponder['name'] . '</option>';
-				}
-			}
-			?>
-		</select>
-		<?php endif; ?>
-		<?php
 
+		<div class="smaily">
+			<!-- Title. -->
+			<div class="section">
+				<h2>
+					<?php esc_attr_e( 'Title', 'smaily' ); ?>
```

Comment:
I did a little digging on translation function usage and found that the correct approach would be to use either `esc_attr__`/`esc_attr_e` or `esc_html__`/`esc_html_e`. Former for usage in the HTML element attributes, and latter for usage in HTML element text.

Make sure you are using correct function where applicable. For example here, you should be using `esc_html_e` function.